src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1720,7 +1720,7 @@
     <value>An assembly has to opt into preview features before using them.</value>
   </data>
   <data name="DetectPreviewFeaturesMessage" xml:space="preserve">
-    <value>Using '{0}' requires opting into preview features. See https://aka.ms/dotnet-warnings/preview-features for more information.</value>
+    <value>Using '{0}' requires opting into preview features.</value>
```

Comment:
Why not update the messages with required format? Like: `{2} Using '{0}' requires opting into preview features. See {1} for more information.` Maybe it should divided into to resource string:

 - if there is a message : `{2} Using '{0}' requires opting into preview features. See {1} for more information.`
 - if no message : `Using '{0}' requires opting into preview features. See {1} for more information.`